src/nvim/syntax.h

```diff
@@ -2,8 +2,8 @@
 #define NVIM_SYNTAX_H
 
 #include <stdbool.h>
-
 #include "nvim/buffer_defs.h"
```

Comment:
system includes are separated from "local" includes by a newline, so the newline shouldn't be deleted